packages/cloudflare/package.json

```diff
@@ -28,6 +28,16 @@
 			"import": "./dist/api/*.js",
 			"types": "./dist/api/*.d.ts",
 			"default": "./dist/api/*.js"
+		},
+		"./lib/*": {
```

Comment:
I know I suggested `lib` but by looking at the PR it looks to me like we might as well stick with `api` since that's already used 🙂